src/privateJrPlatform/multiTenancy.ts

```diff
@@ -0,0 +1,37 @@
+const location = typeof window !== 'undefined' ? window.location : undefined
+
+export function getJrPlatformInstanceId(): string {
+  if (!isMultitenancyEnabled()) return import.meta.env.SCRIVITO_TENANT
+
+  if (!location) throw new Error('Could not determine tenant!')
+
+  const tenantFromUrl = location.pathname.match(/^\/([0-9a-f]{32})\b/)?.[1]
+  const tenantFromQuery = new URLSearchParams(location.search).get('tenantId')
+  const tenant = tenantFromUrl || tenantFromQuery
+
+  if (!tenant) throw new Error('Could not determine tenant!')
+
+  return tenant
+}
+
+export function getJrPlatformBaseAppUrl(): string {
+  if (!origin) throw new Error('No origin defined!')
```

Comment:
* [ ] `PRIVATE_JR_PLATFORM=true npm run prerender` fails here b/c of the missing `window !== 'undefined'` fallback to SCRIVITO_ORIGIN.